src/interface/blas3_interface.hpp

```diff
@@ -160,6 +160,421 @@ typename executor_t::policy_t::event_t _gemm_batched(
                        _ldb, _beta, _C, _ldc, batch_size, batch_type);
 }
 
+/**
+ * @brief Implementation of Triangle Solve with Multiple Right Hand Sides
+ * (TRSM).
+ * @param Side Indicates if A is on the left or right of X
+ * @param Triangle If A is lower or upper triangular
+ * @param Transpose indicates the form that the matrix A will take in the
+ * multiplication
+ * @param Diagonal If A has a unit or non-unit diagonal
+ * @param M The number of rows in of matrix B, must be at least 0
```

Comment:
```suggestion
 * @param M The number of rows in of matrix B, must be at least 1
```

I believe `invalid_argument` is thrown if it's 0. The same for the other parameters.

Reply:
Fixed